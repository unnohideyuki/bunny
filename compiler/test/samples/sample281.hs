main = do
  print $ (10::Double) / 3
  print $ (10::Float) / 3
  print $ recip (3::Double)
  print $ recip (3::Float)

