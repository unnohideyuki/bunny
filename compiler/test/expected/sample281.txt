3.3333333333333335
3.3333333
0.3333333333333333
0.33333334
